fluentlenium-core/src/main/java/org/fluentlenium/utils/chromium/ChromiumControl.java

```diff
@@ -0,0 +1,13 @@
+package org.fluentlenium.utils.chromium;
+
+/**
+ * Control interface for chromium api.
+ */
+public interface ChromiumControl {
+    /**
+     * Features related to CSS loaded in the active page.
+     *
+     * @return a new chromium api instance
+     */
+    ChromiumApi getChromiumApi();
```

Comment:
Maybe ChromiumApi can be singleton?

Reply:
Probably it can be, but I don't really see any advantages.

Reply:
Probably it can, but I fail to see the benefit of doing this.